### PR TITLE
Editorial update of "API Identifier" rule

### PR DESCRIPTION
**Pull request by tfrauenstein**

### CODE REVIEW

**Comment by maxim-tschumak** [comment]
:+1:

File: chapters/meta-information.adoc [hunk 1]
```diff
@@ -58,11 +58,16 @@ Example:
 ----
 
 [#215]
-== {SHOULD} Provide API Identifiers to Support API Histories
+== {SHOULD} Provide API Identifiers
 
 Each API should be identified by an explicit, owner assigned, globally unique,
-and immutable API identifier, that allows to reveal the history and evolution
+and immutable API identifier. APIs evolve and every API aspect may change, 
+but the API identifier not. Based on the API identifier, we can track the
```
Comment (whiskeysierra) [inline chapters/meta-information.adoc]: > [..] but the API identifier may not.

Comment (whiskeysierra) [reply chapters/meta-information.adoc]: > [..] except the API identifier.

Comment (tfrauenstein) [reply chapters/meta-information.adoc]: 👍

File: chapters/meta-information.adoc [hunk 2]
```diff
@@ -58,11 +58,16 @@ Example:
...
+== {SHOULD} Provide API Identifiers
 
 Each API should be identified by an explicit, owner assigned, globally unique,
-and immutable API identifier, that allows to reveal the history and evolution
+and immutable API identifier. APIs evolve and every API aspect may change,
+except the API identifier. Based on the API identifier, we can track the 
+API life cycle and manage the history and evolution
 of an API as a sequence of API specifications.
+For instance, we can provide our API Linter https://github.com/zalando/zally[API Linter Zally]
+with capability to check compatibility and correct semantic versioning
```
Comment (mfellner) [inline chapters/meta-information.adoc]: with **a** capability check

File: chapters/meta-information.adoc [hunk 3]
```diff
@@ -58,11 +58,16 @@ Example:
...
 
 Each API should be identified by an explicit, owner assigned, globally unique,
-and immutable API identifier, that allows to reveal the history and evolution
+and immutable API identifier. APIs evolve and every API aspect may change,
+except the API identifier. Based on the API identifier, we can track the 
+API life cycle and manage the history and evolution
 of an API as a sequence of API specifications.
+For instance, we can provide our API Linter https://github.com/zalando/zally[API Linter Zally]
+with capability to check compatibility and correct semantic versioning 
+of an API update draft against API running in production.
```
Comment (mfellner) [inline chapters/meta-information.adoc]: against **an** API

File: chapters/meta-information.adoc (hunk 2 above)
Comment (tfrauenstein) [reply chapters/meta-information.adoc]: 👍

File: chapters/meta-information.adoc (hunk 3 above)
Comment (tfrauenstein) [reply chapters/meta-information.adoc]: 👍

File: chapters/meta-information.adoc (hunk 2 above)
Comment (tfrauenstein) [reply chapters/meta-information.adoc]: 👍

**Comment by mfellner** [comment]
👍

**Comment by tkrop** [comment]
👍

**Pull request merged**